integreat_cms/static/src/js/forms/prevent-premature-submission.ts

```diff
@@ -0,0 +1,26 @@
+export default class SubmissionPrevention {
```

Comment:
Maybe add some short clarification here? Because from the code itself it is rather difficult to understand why it was added